services/QuillLMS/spec/queries/progress_reports/district_concept_reports_spec.rb

```diff
@@ -3,37 +3,32 @@
 require 'rails_helper'
 
 describe ProgressReports::DistrictConceptReports do
-  describe '#results' do
-    let!(:school) { create(:school) }
-    let!(:teacher) { create(:teacher) }
-    let!(:admin) { create(:teacher) }
-    let!(:classroom) { create(:classroom) }
-    let!(:student) { create(:student) }
-    let!(:schools_admins) { create(:schools_admins, school: school, user: admin) }
-    let!(:schools_users) { create(:schools_users, school: school, user: teacher) }
-    let!(:classrooms_teacher) { create(:classrooms_teacher, user: teacher, role: "owner", classroom: classroom) }
-    let!(:classroom_unit) { create(:classroom_unit, classroom: classroom, assigned_student_ids: [student.id]) }
-    let!(:activity_session) { create(:activity_session_without_concept_results, classroom_unit: classroom_unit, user_id: student.id) }
-    let!(:concept_result) { create(:concept_result, activity_session: activity_session) }
+  context 'integration', :integration do
+    let(:sample_prod_admin_id) { 3737095 }
+    let(:expected_row_keys) do
+      %w(school_name teacher_name classroom_name student_name correct incorrect percentage)
+    end
+
+    around do |a_spec|
```

Comment:
I wanted this to be a true integration test, so I'm bypassing VCR. I mock BigQuery responses in a following file, for unit style testing.

Reply:
How is circleci picking up  'BIGQUERY_CREDENTIALS'

Reply:
CircleCI doesn't see this file due to the `integration` filter. However, you can set up secret env vars in CircleCi in `Project > Settings`, same pattern as Heroku.

Reply:
Good point